### DISCUSSION TITLE
saf-training/courses/user/04

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#4] [Ideas]
# saf-training/courses/user/04

The MITRE Security Automation Framework Team's Introduction to InSpec profile development

https://mitre.github.io/saf-training/courses/user/04.html

**Comment by NTPowers** [comment]
Please make saf@groups.mitre.org clickable.

**Reply by karikarshivani** [reply]
It is now clickable. Thank you for your comment!
